tests/network-tests/src/fixtures/council/ElectCouncilFixture.ts

```diff
@@ -40,6 +64,8 @@ export class ElectCouncilFixture extends BaseQueryNodeFixture {
 
     // Announcing stage
     await this.api.untilCouncilStage('Announcing')
+    const x = await this.api.query.council.announcementPeriodNr()
+    this.debug(`announcement period ${x.toNumber()}`)
```

Comment:
Perhaps your intention here is to debug/check that we are still in the announcing period expected. Maybe assert the condition instead of logging and continuing?

Reply:
I have removed it now